src/ansys/dpf/core/results.py

```diff
@@ -263,7 +263,12 @@ def eval(self):
         >>> fc = disp.on_all_time_freqs.eval()
 
         """
-        fc = self.__call__().outputs.fields_container()
+        outputs = self.__call__().outputs
+        if hasattr(outputs, "fields_container"):
+            fc = outputs.fields_container()
+        else:
+            # Support operators where the first output is not named "field_container"
+            fc = outputs[0]()
```

Comment:
```suggestion
        fc = self().outputs[0]()
```

Reply:
ah wait... no it is not exactly the same thing @rafacanton , in the first case, if there is an output pin in any position called fields_container, we get this one. In the second case, we take the first output pin.
So I guess making this the default would not be retro-compatible.... sadly